umm-spec-lib/test/cmr/umm_spec/test/validation/umm_spec_collection_validation_tests.clj

```diff
@@ -202,8 +202,6 @@
           (h/assert-warnings-multiple-invalid
```

Comment:
This test is to verify multiple validation warnings can be returned. It is no longer testing that since there is only one condition. We should create another validation warning for this (maybe add a second project with a start date in the future).

Reply:
I just looked at this and was writing a fix, I think the test isn't about multiple warnings, its for specifically testing Start and End date being in the past.

Reply:
Oh ok - we should change the assertion to use h/assert-invalid rather than the multiple-invalid one and the testing block to say "start date not in the past".